common-arcana.lic

```diff
@@ -64,19 +65,9 @@ module DRCA
     true
   end
 
-  def buff(spell, settings)
```

Comment:
Were you still planning on leaving `buff` here (which would just call `cast_spell`)?